nix/modules/devShells.nix

```diff
@@ -131,15 +131,15 @@
               '')
 
               (pkgs.writeShellScriptBin "scripts-cargo-update" ''
-                 set -xeu -o pipefail
-
-                 # Update the Holochain project Cargo.lock
-                 cargo update --manifest-path Cargo.toml
-                 # Update the release-automation crate's Cargo.lock
-                 cargo update --manifest-path crates/release-automation/Cargo.toml
-                 # Update the WASM workspace Cargo.lock
-                 cargo update --manifest-path crates/test_utils/wasm/wasm_workspace/Cargo.toml
-               '')
+                set -xeu -o pipefail
```

Comment:
This wasn't deliberate, my IDE did it. It looks correct though because this was indented with 3 spaces?